runtime/battery-station/src/parameters.rs

```diff
@@ -312,6 +312,10 @@ parameter_types! {
     // Orderbook parameters
     pub const OrderbookPalletId: PalletId = ORDERBOOK_PALLET_ID;
 
+    // Parimutuel parameters
+    pub const MinBetSize: Balance = 50 * CENT;
```

Comment:
Are there any benefits of setting `MinBetSize` higher than `ExistentialDeposit`?

```suggestion
    pub const MinBetSize: Balance = ExistentialDeposit::get();
```

Reply:
I would set it higher than the ExistentialDeposit, because just for the fact that creator fees are taken from it, which would drop it below the ExistentialDeposit and therefore doesn't allow to transfer the residue to the pot account if the pot account is still zero.

Reply:
I understand, make sense. Maybe couple it directly to the `ExistentialDeposit` anyways. For example in a way that ensures proper functioning as long as the fees are less than 99% of the exchanged value.

```suggestion
    pub const MinBetSize: Balance = 100 * ExistentialDeposit::get();
```